war/src/main/js/widgets/add/addform.less

```diff
@@ -63,39 +63,26 @@
     position: relative;
     border: 1px solid #cccccc;
     border-top: none;
-    padding: 20px;
     background-color: #f9f9f9;
+    padding:10px;
```

Comment:
This is not an unrelated change. Copy item needs to show selected state, it did not before.